interface/timing/timing.smithy

```diff
@@ -0,0 +1,54 @@
+// endpoint-enumerator-interface.smithy
+
+// Tell the code generator how to reference symbols defined in this namespace
+metadata package = [{
+     namespace: "org.wasmcloud.interface.timing",
+     crate: "wasmcloud-interface-timing"
+ }]
+
+namespace org.wasmcloud.interface.timing
+
+use org.wasmcloud.model#wasmbus
+use org.wasmcloud.model#U32
+
+/// Allows actors to `sleep` for a specified duration, or until a desired time.
+@wasmbus(
+    contractId: "wasmcloud:timing",
+    providerReceive: true,
+)
+
+service Timing {
+    version: "0.1",
+    operations: [ Sleep, SleepUntil, Now ]
```

Comment:
I think it would be best to limit this operation to just `Now`, since sleeping is complicated within wasmCloud and could only be done for a max of just under 2 seconds if using the default RPC timeout